.github/workflows/devcontainer.yml

```diff
@@ -90,17 +90,14 @@ jobs:
           --env LOCAL_WORKSPACE_FOLDER=$(pwd) \
           --env PYTEST_EXTRA_OPTIONS=$PYTEST_EXTRA_OPTIONS \
           --workdir /workspaces/MLOS \
-          --name mlos-devcontainer mlos-devcontainer sleep infinity
+          --name mlos-devcontainer mlos-devcontainer sleep 1800
```

Comment:
The full suite of tests should never run more than 30 minutes I think, so we can let the devcontainer exit after that and throw an error.